### DISCUSSION TITLE
Reset to Factory Prefs

### DISCUSSION THREAD

**Discussion by CipherLawAI** [rxhanson/RectanglePro-Community#392] [Q&A]
Hi. After a bit of playing around with the Rectangle Pro settings (terminal and in the Settings window), I'd like to wipe down the the original factor settings and start over.  On Sonoma 14.1.2, where would I find the files/settings to delete? Sry for the basic question, relatively new to mac.

**Comment by rxhanson** [comment]
Good question. Here is the terminal command:

`defaults delete com.knollsoft.Hookshot`

Then restart the app.

**Comment by CipherLawAI** [comment]
Thx for the reply!  BTW, love the app. Running on MBP 16 driving a 49" curved gaming monitor (for business use). RP is absolutely critical.
